js/src/edit-block/hooks/useCopyToClipboard.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import Clipboard from 'clipboard';
+
+/**
+ * WordPress dependencies
+ */
+import { useRefEffect } from '@wordpress/compose';
+import { useRef } from '@wordpress/element';
+
+function useUpdatedRef( value ) {
+	const ref = useRef( value );
+	ref.current = value;
+	return ref;
+}
+
+/**
+ * Copies the given text to the clipboard when the element is clicked.
+ *
+ * Forked from Gutenberg, with only minor changes.
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/1103f7ba9f20fada5af22cb6d86bd26e75defea6/packages/compose/src/hooks/use-copy-to-clipboard/index.js
```

Comment:
The `@wordpress/compose` package has a hook [useCopyToClipboard()](https://github.com/WordPress/gutenberg/blob/1103f7ba9f20fada5af22cb6d86bd26e75defea6/packages/compose/src/hooks/use-copy-to-clipboard/index.js) with a breaking change.

We don't know which one we'll get, depending on whether Gutenberg is active. 

This isn't ideal, but it forks it.